addons/website_forum/tests/test_controller.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+from odoo.tests.common import HttpCase, new_test_user
+
+
+class TestController(HttpCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        portal_user = new_test_user(cls.env, login='portal_user', groups='base.group_portal')
+        cls.portal = portal_user.login
+        admin_user = new_test_user(cls.env, login='admin_user', groups='base.group_user,base.group_system')
+        cls.admin = admin_user.login
+        cls.headers = {"Content-Type": "application/json"}
+        cls.pixel = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAADElEQVR4nGNgYGAAAAAEAAH2FzhVAAAAAElFTkSuQmCC'
```

Comment:
Nitpicking: do we actually need a string that long for the test?

Reply:
I found a shorter one that works. This string is a base64 code that is later decoded and on which there a type checks to ensure it is an image.

Reply:
Thanks for the explanation, I was asking just to be sure we couldn't just trip that one (so the line is not that long) but if you had to find another one, you could have very well kept this one :)
I didn't mean to make it harder / lose time. (that would have been a valid answer :))
But thank you! Now that is done :+1: